app.py

```diff
@@ -194,6 +199,36 @@ def get_snap_id(snap_name):
     return details_response.json()['snap_id']
 
 
+def calculate_color(thisCountry, maxCountry, maxColor, minColor):
+    countryFactor = float(thisCountry)/maxCountry
+    colorRange = maxColor - minColor
+    return int(colorRange*countryFactor+minColor)
+
+
+def calculate_colors(countries, max_users):
```

Comment:
There is so much different stuff in app.py already. We should split it up into smaller managable pieces at some point.

Reply:
I agree with you, but not in this PR I am planning on doing a refactor of the file next week